RecoHGCal/Configuration/python/RecoHGCal_EventContent_cff.py

```diff
@@ -10,6 +10,7 @@
       'keep *_ticlMultiClustersFromTrackstersTrkEM_*_*',
       'keep *_ticlMultiClustersFromTrackstersMIP_*_*',
       'keep *_ticlMultiClustersFromTrackstersMerge_*_*',
+      'keep *_ticlMultiClustersFromSimTracksters_*_*',
```

Comment:
Do you have a size estimation on what added to AOD, i.e. estimate from ttbar with PU200?

Reply:
@srimanob, the plan is to drop multiclusters completely in the next few months

Reply:
@felicepantaleo 
Thanks for the clarification. If you have a chance for another commit, could you please add a comment in this part. So that we will not forget when we try to review content sometimes later. Thanks.

Reply:
sure, I will add a comment now